### PR TITLE
Get router working in history mode. Split MenuManagerController into View and Service class

### PR DESCRIPTION
**Pull request by Mark-de-Haan**

### CODE REVIEW

**Comment by Mark-de-Haan** [comment]
test this please

File: molgenis-metadata-manager/src/main/frontend/src/store/actions.js [hunk 1]
```diff
@@ -31,15 +31,7 @@ export default {
     // TODO can we filter system entities with REST call??
     get({apiUrl: '/api'}, '/v2/sys_md_EntityType?num=10000')
       .then(response => {
-        const nonSystemEntities = response.items.filter(function (item) {
```
Comment (ConnorStroomberg) [inline molgenis-metadata-manager/src/main/frontend/src/store/actions.js]: would be good to split this off into a separate pull request

File: navigator/pom.xml [hunk 2]
```diff
@@ -29,7 +29,7 @@
                         </goals>
                         <phase>initialize</phase>
                         <configuration>
-                            <nodeVersion>v6.10.0</nodeVersion>
```
Comment (ConnorStroomberg) [inline navigator/pom.xml]: 6 is the LTS version maybe we should discuss this
